.scalafix.conf

```diff
@@ -1,5 +1,5 @@
 rules = [
-  ExplicitResultTypes,
+#  ExplicitResultTypes,
```

Comment:
```
[error] (rules / scalafixAll) scalafix.sbt.InvalidArgument: The ExplicitResultTypes rule only supports the exact Scala versions 'List(2.13.6, 2.13.4, 2.13.5)' for this binary version. To fix this problem, either remove ExplicitResultTypes from .scalafix.conf or change the Scala version of your build to match one of the supported versions.
```

Reply:
make sense. we have to that everytime we upgrade the version!